pkg/authorization/registry/subjectaccessreview/rest.go

```diff
@@ -99,15 +99,14 @@ func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 	requestContext := kapi.WithNamespace(kapi.WithUser(ctx, userToCheck), subjectAccessReview.Action.Namespace)
 	attributes := authorizer.ToDefaultAuthorizationAttributes(subjectAccessReview.Action)
 	allowed, reason, err := r.authorizer.Authorize(requestContext, attributes)
-	if err != nil {
```

Comment:
if there's an internal error, we don't necessarily know whether they are allowed or not... we shouldn't return a definitive answer